pkg/netrasp/sros.go

```diff
@@ -0,0 +1,97 @@
+package netrasp
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// SROS is the Netrasp driver for Nokia SR OS (MD-CLI).
+type sros struct {
+	Connection connection
+}
+
+// Close connection to device.
+func (i sros) Close(ctx context.Context) error {
+	i.Connection.Close(ctx)
+
+	return nil
+}
+
+// Configure device.
+func (i sros) Configure(ctx context.Context, commands []string) (string, error) {
+	var output string
+	_, err := i.Run(ctx, "edit-config exclusive")
+	if err != nil {
+		return "", fmt.Errorf("unable to enter exclusive edit mode: %w", err)
+	}
+	for _, command := range commands {
+		result, err := i.Run(ctx, command)
+		if err != nil {
+			return output, fmt.Errorf("unable to run command '%s': %w", command, err)
+		}
+		output += result
+	}
+	_, err = i.Run(ctx, "commit")
```

Comment:
I seem to remember from Alcatel Timos that there was some "root" command, would anything like that be needed to return to the correct context or anything like that?

Reply:
thanks for reminding me that I technically need to exit from configuration mode to return to the running datastore
this added in 4ee60cb